docs: move the test readme to the official doc (#9226)
rook/rook: Documentation/development-environment.md

---
title: Developer Environment

> weight: 12100

Reviewer: How about moving this before the storage providers doc?
```suggestion
weight: 12030
```

indent: true
---

# Install Kubernetes

You can choose any Kubernetes install of your choice. The test framework only depends on `kubectl`
being configured. To install `kubectl`, please see the [official guide](https://kubernetes.io/docs/tasks/tools/#kubectl).

# Minikube

The developers of Rook are working on Minikube and thus it is the recommended way to quickly get

> Rook up and running. The Rook authors do not recommend using Minikube for production but

Reviewer: ```suggestion
Rook up and running. Minikube should not be used for production but the Rook authors
```
